src/helpers/metrics.ts

```diff
@@ -62,3 +62,7 @@ export const emitMetric = (name: string, body?: any) => {
   });
   uploadMetrics();
```

Comment:
In this function, could we also check if Amplitude tracking is not blocked in the browser? Right now, we're getting quite a few error logs from this.